integration/Substra/flamby-integration-into-substra.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
remove the notebook please (and from all commits with interactive rebasing please)

Reply:
Why removing it from all commit ? And remove the ipynb means we remove the source file that generate the md document... You're sure about removing it ?

Reply:
if @ThibaultFy removes it then squashes the commits before merging, it will not appear in the git history anymore, is it enough?

Reply:
Removing in order to get rid of the heavy diffs induced by the notebooks, which take a lot of space.